pkg/lightning/config/global.go

```diff
@@ -65,6 +65,7 @@ type GlobalMydumper struct {
 type GlobalImporter struct {
 	Backend     string `toml:"backend" json:"backend"`
 	SortedKVDir string `toml:"sorted-kv-dir" json:"sorted-kv-dir"`
+	RemoteAddr  string `toml:"remote-addr" json:"remote-addr"`
```

Comment:
I'm not sure if `remote-addr` is an appropriate name, or if we could use a clearer name, such as remote-load-data-worker-addr?